module/VuFind/src/VuFind/Search/Solr/DeduplicationListener.php

```diff
@@ -89,18 +89,18 @@ class DeduplicationListener
     /**
      * Constructor.
      *
-     * @param BackendInterface        $backend          Search backend
-     * @param ServiceLocatorInterface $serviceLocator   Service locator
-     * @param string                  $searchConfig     Search config file id
-     * @param string                  $dataSourceConfig Data source file id
-     * @param bool                    $enabled          Whether deduplication is
-     * enabled
+     * @param Backend            $backend          Search backend
+     * @param ContainerInterface $serviceLocator   Service locator
+     * @param string             $searchConfig     Search config file id
+     * @param string             $dataSourceConfig Data source file id
+     * @param bool               $enabled          Whether deduplication is
+     *                                             enabled
```

Comment:
Whitespace is not required here; PHPDoc does not require this kind of text alignment, as it reduces readability for long descriptions.